best_fit_fluxes2escher_fluxes_csv.sh

```diff
@@ -1,4 +1,11 @@
 #!/bin/bash
 
 echo "ID,Avg" > $2
-tail -n+2 $1 | gawk -vFPAT='[^,]*|"[^"]*"' '{print $1 "," ($4 + $5) / 2.0 }' >> $2
+FOURTHCOL=`head -n 1 $1 | gawk -vFPAT='[^,]*|"[^"]*"' '{ print $4 }'`
+if [ $FOURTHCOL == "Value" ]; then
```

Comment:
Based on the header of the best fit fluxes file, this will decide whether to use a single value or an average (for backwards compatibility). Is the fourth column name "Value" @cfoguet or we need to change this if?